backend/btrixcloud/basecrawls.py

```diff
@@ -171,14 +171,15 @@ async def get_crawl_out(
                 res["collections"] = await self.colls.get_collection_names(coll_ids)
 
             if res.get("version", 1) == 2:
-                res["initialPages"] = await self.page_ops.list_replay_query_pages(
+                res["initialPages"], _ = await self.page_ops.list_pages(
                     crawl_ids=[crawlid], is_seed=True, page_size=25
                 )
 
                 oid = res.get("oid")
                 if oid:
                     res["pagesQueryUrl"] = (
-                        get_origin(headers) + f"/api/orgs/{oid}/crawls/{crawlid}/pages"
+                        get_origin(headers)
+                        + f"/api/orgs/{oid}/crawls/{crawlid}/pagesSearch"
```

Comment:
I think this may need to be updated in collection replay.json endpoints as well :)

Reply:
Or maybe not, though I can see why I was confused now.

At this point the differences between `crawls/{crawlid}/pages` and `crawls/{crawlid}/pagesSearch` seems to be just that the latter has fewer filters and doesn't include the total... should we have two endpoints at that point, especially if we don't have a `/pagesSearch` equivalent for collections? Whether or not to include the total could just be a query param passed by the frontend...

Reply:
On the other hand, maybe the distinction is worth keeping so that we can have different response models, since returning an incorrect `total` seems worse than omitting it...

I think I could be persuaded either way here, but would be nice for crawl and collection page endpoints to be a bit more symmetrical one way or the other.

Reply:
Ok, I guess we could make both `/pagesSearch` then, and keep the existing `/pages` as is in 1.13. Would require updating the tests but would be no less to backwards compatibility. Perhaps at a later time, can consolidate the two, rather than remove..